deps/zlib/zlib.gyp

```diff
@@ -12,44 +13,148 @@
+              ],
+            }],
+            ['arm_fpu=="neon"', {
+              'defines': [ 'ADLER32_SIMD_NEON' ],
+            }],
+          ],
+          'include_dirs': [ '<(ZLIB_ROOT)' ],
+          'defines': [ 'ZLIB_IMPLEMENTATION' ],
+          'direct_dependent_settings': {
+            'include_dirs': [ '<(ZLIB_ROOT)' ],
+          },
+          'sources': [
+            '<!@pymod_do_main(GN-scraper "<(ZLIB_ROOT)/BUILD.gn" "\\"zlib_adler32_simd\\".*?sources = ")',
+          ],
+        }, # zlib_adler32_simd
+        {
+          'target_name': 'zlib_arm_crc32',
+          'type': 'static_library',
+          'conditions': [
+            ['OS!="ios"', {
+              'conditions': [
+                ['OS!="win" and llvm_version!="0.0"', {
+                  'cflags': [ '-march=armv8-a+crc' ],
+                }],
+                ['OS=="android"', {
+                  'defines': [ 'ARMV8_OS_ANDROID' ],
+                }],
+                ['OS=="linux"', {
+                  'defines': [ 'ARMV8_OS_LINUX' ],
+                }],
+                ['OS=="win"', {
+                  'defines': [ 'ARMV8_OS_WINDOWS' ],
+                }],
+              ],
+              'defines': [ 'CRC32_ARMV8_CRC32' ],
+              'include_dirs': [ '<(ZLIB_ROOT)' ],
+              'direct_dependent_settings': {
+                'include_dirs': [ '<(ZLIB_ROOT)' ],
+              },
+              'sources': [
+                '<!@pymod_do_main(GN-scraper "<(ZLIB_ROOT)/BUILD.gn" "\\"zlib_arm_crc32\\".*?sources = ")',
+              ],
+            }],
+          ],
+          'defines': [ 'ZLIB_IMPLEMENTATION' ],
+        }, # zlib_arm_crc32
+        {
+          'target_name': 'zlib_inflate_chunk_simd',
+          'type': 'static_library',
+          'conditions': [
+            ['(target_arch in "ia32 x64 x32" and OS!="ios")', {
+              'defines': [ 'INFLATE_CHUNK_SIMD_SSE2' ],
+              'conditions': [
+                ['target_arch=="x64"', {
+                  'defines': [ 'INFLATE_CHUNK_READ_64LE' ],
+                }],
+              ],
+            }],
+            ['arm_fpu=="neon"', {
+              'defines': [ 'INFLATE_CHUNK_SIMD_NEON' ],
+              'conditions': [
+                ['target_arch=="arm64"', {
+                  'defines': [ 'INFLATE_CHUNK_READ_64LE' ],
+                }],
+              ],
+            }]
+          ],
+          'include_dirs': [ '<(ZLIB_ROOT)' ],
+          'defines': [ 'ZLIB_IMPLEMENTATION' ],
+          'direct_dependent_settings': {
+            'include_dirs': [ '<(ZLIB_ROOT)' ],
+          },
+          'sources': [
+            '<!@pymod_do_main(GN-scraper "<(ZLIB_ROOT)/BUILD.gn" "\\"zlib_inflate_chunk_simd\\".*?sources = ")',
+          ],
+        }, # zlib_inflate_chunk_simd
+        {
+          'target_name': 'zlib_crc32_simd',
           'type': 'static_library',
+          'conditions': [
+            ['OS!="win" or llvm_version!="0.0"', {
+              'cflags': [
```

Comment:
Is this getting used at all? The latest **test-macOS / test-macOS (pull_request)** logs don't seem to include the `cflags` for this target in https://github.com/nodejs/node/pull/33044/checks?check_run_id=3008235129#step:5:1336:
```console
  cc -o /Users/runner/work/node/node/out/Release/obj.target/zlib_crc32_simd/deps/zlib/crc32_simd.o ../deps/zlib/crc32_simd.c '-DV8_DEPRECATION_WARNINGS' '-DV8_IMMINENT_DEPRECATION_WARNINGS' '-D_GLIBCXX_USE_CXX11_ABI=1' '-D_DARWIN_USE_64_BIT_INODE=1' '-DOPENSSL_NO_PINSHARED' '-DOPENSSL_THREADS' '-DCRC32_SIMD_SSE42_PCLMUL' '-DZLIB_IMPLEMENTATION' -I../deps/zlib  -O3 -gdwarf-2 -mmacosx-version-min=10.13 -arch x86_64 -Wall -Wendif-labels -W -Wno-unused-parameter -fno-strict-aliasing -MMD -MF /Users/runner/work/node/node/out/Release/.deps//Users/runner/work/node/node/out/Release/obj.target/zlib_crc32_simd/deps/zlib/crc32_simd.o.d.raw   -c
```